wds/esbuild-bundle-plugin.js

```diff
@@ -12,6 +12,9 @@ export function esbuildBundlePlugin({ pathsToBundle }) {
           format: 'esm',
           minify: true,
           write: false,
+          // prevents esbuild from bundling types instead of the actual package
+          // because of the `paths` option within the regular `tsconfig.json` file
+          tsconfig: 'src/lib/leaflet/tsconfig-empty.json',
```

Comment:
question: is it possible to just pass an empty object as a `tsconfig` instead of relying on a file?

Reply:
No :sob: (the only other solution would be to use `node:fs` to create a tmp file and remove it once esbuild is done but I figured it wasn't worth it?